test/unit/downloader.test.ts

```diff
@@ -101,14 +101,6 @@ describe('Downloader class', () => {
     await expect(downloader.downloadContent('')).rejects.toThrowError()
   })
 
-  test('downloadContent successfully downloaded an image', async () => {
```

Comment:
Would not have been better to download it from an other API end-point? Or maybe this test is just not useful?

Reply:
Done